### PR TITLE
Add support for Kubernetes 1.12

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```release-note
Kubermatic now supports Kubernetes 1.12
```

### CODE REVIEW

File: addons/canal/daemonset.yaml [hunk 1]
```diff
@@ -28,18 +28,7 @@ spec:
...
-          value: "true"
-          effect: NoSchedule
-        # Allow the pod to run on the master.  This is required for
-        # the master to communicate with pods.
-        - key: node-role.kubernetes.io/master
-          effect: NoSchedule
-        # Mark the pod as a critical add-on for rescheduling.
-        - key: "CriticalAddonsOnly"
-          operator: "Exists"
+        - operator: Exists
```
Comment (mrIncompetent) [inline addons/canal/daemonset.yaml:31]: Does this basically ignore all taints?

Comment (alvaroaleman) [reply addons/canal/daemonset.yaml:31]: Yes

**Review by mrIncompetent — DISMISSED** [review]
Please mind updating the addons image tag

**Comment by alvaroaleman** [comment]
@mrIncompetent I updated the tag, PTAL

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**